pkg/common/common_suite_test.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2019 Tigera, Inc. All rights reserved.
```

Comment:
Could you move this to a folder not named common, it's sort of a go anti pattern. I'd just add this to a package called dns and all dns related items could go there.